python-packages/order_utils/README.md

```diff
@@ -0,0 +1,47 @@
+API documentation hosted at [https://0x.readthedocs.io/projects/order-utils/en/latest/](https://0x.readthedocs.io/projects/order-utils/en/latest/)
+
+## Development
+
+### Edit/test cycle
+
+`pip install -e .[dev]`
+
+`./setup.py build`
+
+`./setup.py build_sphinx`
+
+`./setup.py test`
+
+`./setup.py lint`
+
+`./setup.py clean --all`
+
+See `./setup.py --help-commands` for more info.
```

Comment:
Can we also copy the TS README structure for these? Build, clean, lint are separate sections (https://github.com/0xProject/0x-monorepo/tree/development/packages/order-utils#build)